tests/conftest.py

```diff
@@ -132,6 +132,11 @@ def db_version(request):
     return request.param
 
 
+@pytest.fixture(scope="function", params=[1000000, 3630000])
```

Comment:
shouldn't second param be a constant from `ConsensusConstants`?

Reply:
it could be, but that would add a dependency on the default_constants module here, which I don't think is worth it. my understanding is that all tests would pay for that